src/selectors/per-thread/markers.js

```diff
@@ -383,12 +383,15 @@ export function getMarkerSelectorsPerThread(
   );
 
   /**
-   * This returns only FileIO markers.
+   * This returns only FileIO markers for the header.
+   * Also excludes FileIO markers that belong to other threads.
```

Comment:
I'm slightly confused here: if they belong to other thread, should we display them in these other threads, if we have them profiled?

Reply:
- If a non-main thread is profiled, FileIO markers will appear in their own thread (as you suggest), but they will **also** be duplicated in the main thread! As discussed with Florian and Nazim; the idea is that all IOs can be found on the main thread track (but with some distinction) so the perf person can find them all at one glance, and other threads will contain their own IOs so specific threads can be examined in isolation.
- Non-profiled threads don't have tracks, so we move their IO markers to the main thread for visibility.

Reply:
Yep, agree with Gerald here. Since we also collect the IO markers from the threads we don't profile, it's not possible to visualize them on somewhere else. It probably would be waste of space to create a track for that thread just because it only has some IO markers.

Reply:
Yeah, I don't disagree for the unprofiled threads.
But I'd say that seeing the IO marker twice is a bit of a hack, instead we could easily make a track showing all IO markers from all threads.
This gets the job done but I wonder if this will come back to bite us in the future...

Reply:
What's the advantage of creating a new track for only IO markers?

Reply:
I mean yes, it's not great to have duplicate markers. But in the back-end we wanted to land it like this for now and revisit this decision during the marker API change. The back-end is landed right now but we can make it better incrementally.

Reply:
Yeah sorry I didn't ask that we redo everything now :-) it was more a general comment. Sometimes the best solution is just too long to implement.